packages/cli/src/lib/quire/index.js

```diff
@@ -49,14 +49,34 @@ function getVersion(projectPath) {
   return version
 }
 
+/**
+ * Read the required `quire-11ty` version from starter `package.json` `peerDependencies`
+ *
+ * @param    {String}   projectPath  Absolute system path to the project root
+ *
+ * @return  {String}  version  Quire-11ty semantic version with caret or other
+ * comparators trimmed off the beginning
+ *
+ * @TODO refactor `latest()` function to programmatically return a specific
+ * version of `@thegetty/quire-11ty` from a semantic version string
+ * (i.e `^1.0.0-pre-release.0` => `1.0.0-pre-release.2`) so this string-trimming
+ * logic can be removed
+ */
+function getVersionFromStarterPeerDependencies(projectPath) {
```

Comment:
Though this is very explicit it may be _too_ explicit, for example we might need to `getVersionFromStarter` but not do so from `peerDependencies`, perhaps removing the implementation detail from the method name,
```suggestion
function getVersionFromStarter(projectPath) {
```